### PR TITLE
Jack/debugging

### PR DESCRIPTION
**Pull request by jschluger**
Fixing more bugs from the hit list.

### CODE REVIEW

File: Clicker/ViewControllers/PollBuilderViewController+Extension.swift [hunk 1]
```diff
@@ -10,6 +10,12 @@ import Foundation
 
 extension PollBuilderViewController: PollBuilderViewDelegate {
     
+    func ignoreNextKeyboardHiding() {
+        if isKeyboardShown {
+            shouldIgnoreNextKeyboardHiding = true
+        }
```
Comment (kevinchan159) [inline Clicker/ViewControllers/PollBuilderViewController+Extension.swift]: can shorten this to `shouldIgnoreNextKeyboardHiding = isKeyboardShown`

File: Clicker/ViewControllers/PollsViewController.swift [hunk 2]
```diff
@@ -67,10 +70,14 @@ class PollsViewController: UIViewController {
         
         setupViews()
         setupConstraints()
+        //setupGradient()
```
Comment (kevinchan159) [inline Clicker/ViewControllers/PollsViewController.swift]: can you remove this if we're not using it

File: Clicker/Views/PollBuilder/MCPollBuilderView.swift [hunk 3]
```diff
@@ -181,7 +185,16 @@ extension MCPollBuilderView: PollBuilderMCOptionSectionControllerDelegate {
         let newMCOptionModel = PollBuilderMCOptionModel(type: .newOption(option: "", index: mcOptionModels.count - 1))
         mcOptionModels.insert(newMCOptionModel, at: mcOptionModels.count - 1)
         updateTotalOptions()
-        adapter.reloadData(completion: nil)
+        pollBuilderDelegate.ignoreNextKeyboardHiding()
+        adapter.reloadData { _ in
+            let index = IndexPath(item: 0, section: self.mcOptionModels.count - 2)
```
Comment (kevinchan159) [inline Clicker/Views/PollBuilder/MCPollBuilderView.swift]: what does `self.mcOptionModels.count - 2` refer to?

Comment (jschluger) [reply Clicker/Views/PollBuilder/MCPollBuilderView.swift]: This is because the last cell (at `mcOptionModels.count - 1`) is the add options button, so we want to select the one before that, the last cell for typing an option.

I'll leave a comment in the code to clarify.
